### PR TITLE
Changed name to faster-fifo and modified setup.py

### PR DESCRIPTION
**Pull request by tushartk**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,21 +4,21 @@
...
 ]
 
 setup(
     # Information
-    name='sample-factory',
+    name='faster-fifo',
     version='0.0.1',
-    url='https://github.com/alex-petrenko/sample-factory',
+    url='https://github.com/alex-petrenko/faster-fifo',
     author='Aleksei Petrenko',
```
Comment (alex-petrenko) [inline setup.py:19]: Can you add author='Aleksei Petrenko & Tushar Kumar'?

File: setup.py [hunk 2]
```diff
@@ -4,21 +4,21 @@
...
     license='MIT',
-    keywords='asynchronous reinforcement learning policy gradient ppo impala',
+    keywords='asynchronous ',
```
Comment (alex-petrenko) [inline setup.py:21]: change this to 'multiprocessing data structures'

File: utils.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+"""Utilities."""
```
Comment (alex-petrenko) [inline utils.py:1]: We probably don't need that in the utility package.
I think we should create some FIFO_VERBOSE global variable and unless it is true we should not print/log anything at all (except maybe in tests)
if FIFO_VERBOSE is true, then we should use just standard logger, not colorlog.

Bottom line: we don't want to introduce any unnecessary dependencies for the users of this package

File: setup.py [hunk 4]
```diff
@@ -4,21 +4,21 @@
...
-        include_dirs=['cpp_fast_queue/cpp_lib'],
+        include_dirs=['cpp_faster_fifo/cpp_lib'],
     ),
 ]
 
 setup(
     # Information
-    name='sample-factory',
+    name='faster-fifo',
     version='0.0.1',
```
Comment (alex-petrenko) [inline setup.py:17]: When we're done we should remember to change the version to 1.0.0. Just a note to self

File: setup.py (hunk 1 above)
Comment (tushartk) [reply setup.py]: Made this change

File: setup.py (hunk 2 above)
Comment (tushartk) [reply setup.py]: Made this change too

File: utils.py (hunk 3 above)
Comment (tushartk) [reply utils.py]: Have removed utils and shifted everything to the test file

File: setup.py (hunk 4 above)
Comment (tushartk) [reply setup.py:17]: Sure

**Pull request merged**